website/src/_posts/2017-02-13-stateful-processing.md

```diff
@@ -278,7 +278,7 @@ new DoFn<KV<MyKey, MyValue>, KV<Integer, KV<MyKey, MyValue>>>() {
 
   // A state cell holding a single Integer per key+window
   @StateId("index")
-  private final StateSpec<ValueState<Integer>> indexSpec = 
+  private final StateSpec<ValueState<Integer>> indexSpec =
```

Comment:
The first change to the file should be a bold and loud header about the blog post being updated.

Reply:
Done.